io.gitlab.librewolf-community.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "io.gitlab.librewolf-community",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
```

Comment:
Do you really need the GNOME SDK? Isn't Freedesktop-sdk enough?

Reply:
Oh yeah, looks like Firefox uses freedesktop.Sdk as well: https://hg.mozilla.org/mozilla-central/file/tip/taskcluster/docker/firefox-flatpak/runme.sh#l90.